app/config/services.yml

```diff
@@ -249,16 +249,6 @@ services:
           - '@router'
           - '@security.http_utils'
 
-    elife.journal.security.voter.feature.can_authenticate:
```

Comment:
The voter is removed because the other voter for `FEATURE_CAN_USE_HYPOTHESIS` subsumes it